massmailer/views.py

```diff
@@ -163,6 +173,11 @@ def post(self, request, *args, **kwargs):
             }
             context[results.model_name] = object
             data['render'] = template.full_preview(context)
+            data['render']['header'] = ''
+            if template.is_mailing == 'true':
+                data['render']['header'] = getattr(
```

Comment:
Did you mean to render the actual header ie. `List-Unsubscribe: foo`? Right now you just pass the URL with no context.